io.github.Archeb.opentrace.metainfo.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<component type="desktop">
+  <id>io.github.Archeb.opentrace</id>
+  <name>OpenTrace</name>
+  <summary>OpenTrace is a cross-platform GUI wrapper for NextTrace. Bringing you a sense of familiarity.</summary>
+  <summary xml:lang="zh-Hans">OpenTrace 是 NextTrace 的跨平台 GUI 界面，带来您熟悉但更强大的用户体验。</summary>
+  <developer_name>Archeb</developer_name>
+  <launchable type="desktop-id">io.github.Archeb.opentrace.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LicenseRef-proprietary=https://github.com/Archeb/opentrace#GPL-3.0-1-ov-file</project_license>
+  <description>
+    <p>OpenTrace 是 NextTrace 的跨平台 GUI 界面，带来您熟悉但更强大的用户体验。</p>
+    <p></p>
+    <p>OpenTrace is a cross-platform GUI wrapper for NextTrace. Bringing you a sense of familiarity.</p>
+    <p></p>
+    <p>Note: This build uses a dirty approach to run program with cap_net_raw caps. You'll need to allow pkexec to setcap at the first use.</p>
+    <p>If map isn't showing up, try switching to Google maps in File->Preferences.</p>
+  </description>
+  <screenshots>
+    <screenshot type="default">
+      <image type="source">https://github.com/leohearts/flathub/blob/io.github.Archeb.opentrace/screenshot.png?raw=true</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://i.imgur.com/MxSHIcU.png</image>
+    </screenshot>
+    <screenshot>
+      <image type="source">https://i.imgur.com/brB08tl.png</image>
```

Comment:
imgur isn't a reliable source, screenshots should be from upstream, if not add it here.

Reply:
Ok, I'll delete these.